pkg/scheduler/api/device_info.go

```diff
@@ -53,6 +53,14 @@ func (g *GPUDevice) getUsedGPUMemory() uint {
 	return res
 }
 
+// isIdleGPU check if the device is idled.
+func (g *GPUDevice) isIdleGPU() bool {
+	if g.PodMap == nil || len(g.PodMap) == 0 {
+		return true
+	}
+	return false
+}
+
```

Comment:
Just a small advice, the following expression maybe more brief.
```go
// isIdleGPU check if the device is idled.
func (g *GPUDevice) isIdleGPU() bool {
    return g.PodMap == nil || len(g.PodMap) == 0
}
```
It is not something matters.